### PR TITLE
Change email addresses to placeholders

### PR DESCRIPTION
**Pull request by at0dd**
Issue #26

### CODE REVIEW

File: .env.config [hunk 1]
```diff
@@ -14,11 +14,11 @@ JWT_SECRET='shhhh super secret code here bro'
 ROOT_URL='http://localhost:3000'
 
 # Credentials for the admin user created at app initialization
-ADMIN_EMAIL='admin@hackmit.org'
+ADMIN_EMAIL='admin@example.org'
 ADMIN_PASS='party'
 
 # Used to send verification, registration, and confirmation emails
-EMAIL_CONTACT='HackMIT Team <hackmit.reg.system@gmail.com>'
+EMAIL_CONTACT='HackMIT Team <team@example.com>'
```
Comment (ehzhang) [inline .env.config]: Maybe change this to also remove "HackMIT" and replace it with something like "Hackathon Team"?

File: app.json [hunk 2]
```diff
@@ -16,12 +16,12 @@
...
         "ADMIN_EMAIL": {
             "description": "Credentials for the admin user created at app initialization",
-            "value": "admin@hackmit.org"
+            "value": "admin@example.org"
         },
         "ADMIN_PASS": "party",
         "EMAIL_CONTACT": {
             "description": "Used to send verification, registration, and confirmation emails",
-            "value": "HackMIT Team <hackmit.reg.system@gmail.com>"
+            "value": "HackMIT Team <team@example.com>"
```
Comment (ehzhang) [inline app.json]: Same as above

**Comment by at0dd** [comment]
@ehzhang updated!

**Comment by jlin816** [comment]
Awesome, thanks @at0dd! 😄

**Pull request merged**